### PR TITLE
CASMCMS-8069: Barebones boot test should not disobey --xname argument

### PR DESCRIPTION
**Pull request by mharding-hpe**
## Summary and Scope

See source PR for details:
https://github.com/Cray-HPE/cms-tools/pull/42

## Issues and Related PRs

https://github.com/Cray-HPE/csm/pull/1059
https://github.com/Cray-HPE/docs-csm/pull/2108

## Testing

See source PR for details:
https://github.com/Cray-HPE/cms-tools/pull/42

## Risks and Mitigations

Low risk. Small change.

## Pull Request Checklist

- [X] License file intact
- [X] Target branch correct
- [X] Testing is appropriate and complete, if applicable

### CODE REVIEW

**Comment by mharding-hpe** [comment]
When this merges, its other two manifest PRs should merge as well:
https://github.com/Cray-HPE/docs-csm/pull/2108
https://github.com/Cray-HPE/csm/pull/1059

File: repos/cray.repos [hunk 1]
```diff
@@ -2,6 +2,7 @@ https://arti.dev.cray.com/artifactory/cos-rpm-stable-local/release/cos-2.3/sle15
 
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp3/                                        cray-algol60-csm-rpms-stable-sle-15sp3            --no-gpgcheck -p 88                   cray/csm/sle-15sp3
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/                                        cray-algol60-csm-rpms-stable-sle-15sp2            --no-gpgcheck -p 89                   cray/csm/sle-15sp2
+https://artifactory.algol60.net/artifactory/csm-rpms/hpe/unstable/sle-15sp3/                                      cray-algol60-csm-rpms-unstable-sle-15sp3          --no-gpgcheck -p 87                   cray/csm/sle-15sp3-unstable
```
Comment (rustydb) [inline repos/cray.repos:5]: Unsigned RPMs will not install if I remember correctly, and we do not want to add/maintain unstable repositories.

**Review by rustydb — CHANGES_REQUESTED** [review]
No unstable repositories.

File: repos/cray.repos (hunk 1 above)
Comment (mharding-hpe) [reply repos/cray.repos:5]: Building a stable artifact. Will update these PRs when ready.

File: packages/node-image-kubernetes/base.packages [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #   package_name=version
 # The version is the same version reported by the OS package manager (e.g. zypper).
 
-cray-cmstools-crayctldeploy=1.4.0-1
+cray-cmstools-crayctldeploy=1.5.0-alpha.4+d52e21a
```
Comment (mharding-hpe) [inline packages/node-image-kubernetes/base.packages:6]:
```suggestion
cray-cmstools-crayctldeploy=1.5.0-beta.1
```

File: repos/cray.repos (hunk 1 above)
Comment (mharding-hpe) [reply repos/cray.repos]: Stable RPMs building now. I've updated these PRs, but the artifacts haven't been pushed to artifactory yet. Will comment again when they have.

Comment (mharding-hpe) [reply repos/cray.repos]: Stable artifacts published. This PR is now ready to merge.

**Comment by mharding-hpe** [comment]
> No unstable repositories.

Fixed.

**Pull request merged**